src/main/java/com/github/seanroy/plugins/DeployLambdaMojo.java

```diff
@@ -403,4 +404,10 @@ private String getBucket() {
         }
         return s3Bucket;
     }
+    
+    private String getFileName() {
+        String pattern = Pattern.quote(File.separator);
+        String[] pieces = functionCode.split(pattern);
+        return pieces[pieces.length - 1];
+    }
 }
```

Comment:
Why is this necessary?  Shouldn't fileName have already been set by initFileName?